### PR TITLE
[DSW] [33] Better documentation, misc fixes

### PR DESCRIPTION
**Pull request by scottsand-db**
- improve javadoc
- remove 2 unused files
- better document Operation class constructors
- typo fix

### CODE REVIEW

File: standalone/src/main/java/io/delta/standalone/Operation.java [hunk 1]
```diff
@@ -264,19 +264,37 @@ public String toString() {
...
+    /**
+     * @param name  The {@link Name} of the operation.
+     */
     public Operation(@Nonnull Name name) {
         this(name, Collections.emptyMap(), Collections.emptyMap(), Optional.empty());
     }
 
+    /**
+     * @param name  The {@link Name} of the operation.
+     * @param parameters  Any additional operation parameters, where values are JSON-encoded.
```
Comment (allisonport-db) [inline standalone/src/main/java/io/delta/standalone/Operation.java:276]: nit: "any relevant operation parameters"

File: standalone/src/main/java/io/delta/standalone/expressions/In.java [hunk 2]
```diff
@@ -25,8 +25,8 @@
 import io.delta.standalone.internal.expressions.Util;
 
 /**
- * Usage: {@code new In(expr, exprList)} - Returns true if `expr` is equal to any in `exprList`,
- * else false.
+ * Usage: {@code new In(expr, exprList)} - Returns true if {@code expr}is equal to any in
```
Comment (allisonport-db) [inline standalone/src/main/java/io/delta/standalone/expressions/In.java:28]: nit: space

File: standalone/src/main/java/io/delta/standalone/storage/LogStore.java [hunk 3]
```diff
@@ -89,9 +90,9 @@ public LogStore(Configuration initHadoopConf) {
      * :: DeveloperApi ::
      *
      * Write the given `actions` to the given `path` with or without overwrite as indicated.
-     * Implementation must throw [[java.nio.file.FileAlreadyExistsException]] exception if the file
-     * already exists and overwrite = false. Furthermore, if isPartialWriteVisible returns false,
-     * implementation must ensure that the entire file is made visible atomically, that is,
+     * Implementation must throw {@link java.nio.file.FileAlreadyExistsException} exception if the
+     * file already exists and overwrite = false. Furthermore, if isPartialWriteVisible returns
```
Comment (allisonport-db) [inline standalone/src/main/java/io/delta/standalone/storage/LogStore.java:94]: nit: should fields be formatted as code?

File: standalone/src/main/java/io/delta/standalone/storage/LogStore.java [hunk 4]
```diff
@@ -100,6 +101,7 @@ public LogStore(Configuration initHadoopConf) {
      * @param actions  actions to be written
      * @param overwrite  if true, overwrites the file if it already exists
      * @param hadoopConf  the latest hadoopConf
+     * @throws FileAlreadyExistsException if the file already exists and overwrite is {@code false}
```
Comment (allisonport-db) [inline standalone/src/main/java/io/delta/standalone/storage/LogStore.java:104]: nit: {@code overwrite} (and {@code false} is inconsistent)

File: standalone/src/main/java/io/delta/standalone/types/StructType.java [hunk 5]
```diff
@@ -108,7 +108,9 @@ public StructType add(String fieldName, DataType dataType) {
      *     .add("b", new LongType(), false)
      *     .add("c", new StringType(), true)
      * }</pre>
-     * @param fieldName  The name of the new field to add.
+     * @param fieldName  The name of the new field.
+     * @param dataType  The datatype for the new field.
+     * @param nullable Whether or not the new field is nullable.
```
Comment (allisonport-db) [inline standalone/src/main/java/io/delta/standalone/types/StructType.java:113]: nit: space

**Review by allisonport-db — APPROVED** [review]
LGTM w/minor nits.

File: standalone/src/main/java/io/delta/standalone/storage/LogStore.java (hunk 3 above)
Comment (scottsand-db) [reply standalone/src/main/java/io/delta/standalone/storage/LogStore.java:94]: We don't follow a strict guide here. Whatever seems readable + reasonable

**Pull request merged**